src/main/java/com/internet/shop/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.exceptions.AuthenticationException;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.models.User;
+import com.internet.shop.service.AuthenticationService;
+import com.internet.shop.service.UserService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private UserService userService;
+
+    @Override
+    public User login(String login, String password) throws AuthenticationException {
+        User user = userService.getByLogin(login).orElseThrow(() ->
```

Comment:
I can't see changes in UserService and UserDao classes in this PR

Reply:
I implemented this method when I did HW 26.1